go.mod

```diff
@@ -70,3 +69,5 @@ require (
 	k8s.io/apimachinery v0.0.0-20180821005732-488889b0007f
 	k8s.io/client-go v8.0.0+incompatible
 )
+
+replace github.com/vmware/vic => github.com/hashicorp/vic v1.5.1-0.20190215174116-b15179225d21
```

Comment:
Will this work? I don't see your previous changes in `hashicorp/vic`

Reply:
I thought that `replace` only works for the "main module" aka the thing being built (like `consul` or `nomad`). From https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive:
> As described in the 'go.mod' concepts section above, replace directives provide additional control in the top-level go.mod for what is actually used to satisfy a dependency found in the Go source or go.mod files, **while replace directives in modules other than the main module are ignored when building the main module.**

That bolded part above implies that you would need to copy this replace snippet into the `go.mod` files of everything that directly or indirectly uses `go-discover`

Reply:
good point @rboyer  - I confirmed the behavior you referenced too.  I guess we must use reference the fork.

Reply:
Updated to "hard link" vmware/vic instead.  Thanks for catching this.